Islandora-Devops/claw-playbook
# Adding cantaloupe🍈 to playbook

## Ticket
Resolves: https://github.com/Islandora-CLAW/CLAW/issues/694

## Caveats 
Doesn't include any configuration for openseadragon, as the original shell script does. I'm going to punt this to https://github.com/Islandora-CLAW/CLAW/issues/693, since that configuration can't be set until that module is installed.

Relevant line here:
https://github.com/Islandora-CLAW/claw_vagrant/blob/master/scripts/cantaloupe.sh#L35

## Testing

- Checkout this patch
- Provision just cantaloupe and tomcat:

```bash
$ vagrant up --no-provision
$ ansible-playbook -i inventory/vagrant bootstrap.yml
$ ansible-playbook -i inventory/vagrant playbook.yml --tags cantaloupe,tomcat8,openjpeg
$ vagrant ssh
$ mkdir -p /var/www/html/drupal/web/
```

- Put test images in `/var/www/html/drupal/web` (tiff, jp2, etc)
- Check that you can see them at: `http://localhost:8080/cantaloupe/iiif/2/FILE.NAME/full/full/0/default.jpg`
- Check logs at: `/var/log/cantaloupe`
